bedrock/tabzilla/templates/tabzilla/tabzilla.js

```diff
@@ -750,7 +702,7 @@ var Tabzilla = (function (Tabzilla) {
     + '        <li><h2>{{ _('Products')|js_escape }}</h2>'
     + '          <div>'
     + '            <ul>'
-    + '              <li><a href="https://www.mozilla.org/firefox/products?icn=tabz">Firefox</a></li>'
+    + '              <li><a href="https://www.mozilla.org/firefox/?icn=tabz">Firefox</a></li>'
```

Comment:
The removed/changed URL is correct. Please revert this href back to `https://www.mozilla.org/firefox/products?icn=tabz`.